src/asset-management/AaveManager.sol

```diff
@@ -168,6 +168,15 @@ contract AaveManager is IAssetManager, Ownable, ReentrancyGuard
         _adjustManagement(lPair, lAmount0Change, lAmount1Change);
     }
 
+    function returnAsset(address aToken, uint256 aAmount) external {
+        require(aAmount > 0, "AM: ZERO_AMOUNT_REQUESTED");
+        IAssetManagedPair lPair = IAssetManagedPair(msg.sender);
+        int256 lAmount0Change = -int256(aToken == lPair.token0() ? aAmount : 0);
+        int256 lAmount1Change = -int256(lAmount0Change == 0 ? aAmount: 0);
+        assert(lAmount0Change <= 0 && lAmount1Change <= 0);
```

Comment:
Can we add a task to turn the SMTSolver on so these assertions can be proven

Reply:
Okay. should we only run it locally or should it be built into the CI too?